src/views/settings/EditPlayerDsp.vue

```diff
@@ -10,6 +10,49 @@
       <v-toolbar-title>{{
         $t("settings.dsp.configure_on", { name: playerName })
       }}</v-toolbar-title>
+      <v-spacer />
+      <v-menu offset-y transition="slide-y-transition">
+        <template v-slot:activator="{ props }">
+          <v-btn
+            v-bind="props"
+            class="mr-4"
+            :class="getButtonClass()"
+            prepend-icon="mdi-tray-arrow-down"
+          >
+            {{ $t("settings.dsp.presets.load") }}
```

Comment:
```suggestion
            <span class="d-none d-md-inline">
              {{ $t("settings.dsp.presets.load") }}
            </span>
```

Show just the icon for smaller screens like phones.